src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -91,7 +98,20 @@ public TlsDetails create(final SSLEngine sslEngine) {
     }
 
     /**
-     * Close this client.
+     * Creates High Level Rest Client for SDK.
+     * @param hostAddress The address of OpenSearch cluster, client can connect to
+     * @param port The port of OpenSearch cluster
+     * @return SDKClient which is internally an RestHighLevelClient. The user is responsible for calling {@link #doCloseRestClient()} when finished with the client
```

Comment:
See above comments on SDKClient usage. Also the linked doClose method should be for HLRC, not Rest Client.  Also, the one that's just "RestClient" should probably be Java Client.